email-service/src/main/java/org/java/mentorship/email/EmailApplication.java

```diff
@@ -7,5 +7,7 @@
 @SpringBootApplication
 public class EmailApplication {
 
-    public static void main(String[] args) { ApplicationContext ctx = SpringApplication.run(EmailApplication.class, args); }
+    public static void main(String[] args) {
+        ApplicationContext ctx = SpringApplication.run(EmailApplication.class, args);
```

Comment:
Don't need this local variable

Just do

 public static void main(String[] args) {
     SpringApplication.run(EmailApplication.class, args);